extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -69,6 +69,12 @@
     @ConfigItem(defaultValue = "10s")
     public Duration connectionTimeout = Duration.ofSeconds(10);
 
+    /**
+     * The maximum size of the connection pool used by the WebClient
+     */
+    @ConfigItem
+    public Optional<Integer> webClientMaxPoolSize;
```

Comment:
It should be `OptionalInt`, I just saw it in Dev Service fixed port configuration :-)

Reply:
@glefloch Please also add `getter/setter` - we use this pattern for all OIDC config properties - to make it easier when creating dynamic OidcTenantConfig